src/components/Deliverable/DeliverableTrackline.tsx

```diff
@@ -255,9 +251,7 @@ function DeliverableTrackLine(props: DeliverableTargetLineProps) {
 							],
 						},
 					});
-				} catch (err) {
-					throw err;
-				}
+				} catch (err) {}
```

Comment:
log w/e error we are getting here or mention why we are not logging it For error logging dont use console.log, instead use console.error